docs/api/app.md

```diff
@@ -432,6 +432,10 @@ app.exit(0)
 
 Returns `Boolean` - `true` if Electron has finished initializing, `false` otherwise.
 
+### `app.whenReady()`
+
+Returns `Promise` - fulfilled when Electron is initialized.
```

Comment:
Maybe also mention that this is an alternative to `app.isReady()` and `app.once('ready')`. Could be confusing to people to know the difference.